mlflow/metrics/metric_definitions.py

```diff
@@ -17,16 +17,17 @@ def standard_aggregations(scores):
     }
 
 
-def _validate_text_data(data, metric_name, column_name):
-    """Validates that the data is text and is non-empty"""
-    if len(data) == 0:
+def _validate_text_data(data, metric_name):
```

Comment:
The column_name is important here, since we need to support: `_validate_text_data(targets, "rouge1")` where the column is not called predictions or model_output.

We can do something like this:
`_validate_text_data(predictions, "rouge2", "prediction or model_output")` if that helps?

Reply:
🤦 will do a similar fix, thanks for catching this